metrics-otlp/README.md

```diff
@@ -0,0 +1,134 @@
+   
+   As this instructions are not providing any backend, the following command sets the backend configuration with placeholders `myEndpoint` and `myToken` which needs to get adjusted to your needs. Be aware of, that a token should be provided using a Secret which gets mounted via the `extraEnvs` parameter. As this cannot be passed via the command itself in an easy way, please consider using a dedicated additional values.yaml file for that.
+
+   ```bash
+   helm upgrade metrics-gateway open-telemetry/opentelemetry-collector --version 0.47.0 --install --namespace $KYMA_NS \
+     -f https://raw.githubusercontent.com/kyma-project/examples/main/metrics-otlp/metrics-gateway-values.yaml \
+     --set config.exporters.otlp.endpoint="{myEndpoint}" \
+     --set config.exporters.otlp.headers.Authorization="Bearer {myToken}"
+   ```
+
+1. Verify the deployment
+   Check that the related Pod has been created in the Namespace and is in the `Running` state:
+   ```bash
+   kubectl -n $KYMA_NS rollout status deploy metrics-gateway
+   ```
+
+## Install the agent
+
+1. Deploy the agent
+
+   The gateway deployed in the previous step can be used to push metrics via OTLP. If you prefer to use the prometheus pull-approach, a dedicated instance is needed for scraping your workload. Furthermore, an agent is required running as a daemonset in order to retrieve node specific metrics.
+   
+   The following command will deploy an otel-collector using the upstream helm chart with prepared [values](./metrics-agent-values.yaml). The values file is defining a metrics pipeline for scraping workload by annotation and pushing them to the gateway. It defines also a second metrics pipeline determining node-specific metrics for your workload from the nodes kubelet and the nodes filesystem itself.
+
+   ```bash
+   helm upgrade metrics-agent open-telemetry/opentelemetry-collector --version 0.47.0 --install --namespace $KYMA_NS \
+     -f https://raw.githubusercontent.com/kyma-project/examples/main/metrics-otlp/metrics-agent-values.yaml \
+     --set config.exporters.otlp.endpoint=metrics-gateway.$KYMA_NS:4317
+   ```
+
+1. Verify the deployment
+   Check that the related Pod has been created in the Namespace and is in the `Running` state:
+   ```bash
+   kubectl -n $KYMA_NS rollout status daemonset metrics-agent
+   ```
+
+## Usage
+
+After you have followed the instructions you should have a setup in place as outlined in the diagram on top of the tutorial, with a gateway and an agent. By default, the node specific metrics are getting ingested by the agent automatically and pushed via the gateway to the configured backend. Furthermore, metrics for the gateway and agent instances itself will be exported.
+
+To add metrics ingestion for your custom workload you have two options:
```

Comment:
```suggestion

## Usage

To add metrics ingestion for your custom workload, you have the following options:
```

if we don't mention "two", we won't have to update this line if another option is added.